src/main/resources/com/google/api/codegen/ruby/method_sample.snip

```diff
@@ -16,12 +16,8 @@
 @end
 
 @snippet sampleCodeCore(apiMethod)
-  @join fileImport : apiMethod.initCode.importSection.appImports
-    @join alias : fileImport.types
-      {@alias.nickname} = {@alias.fullName}
-    @end
-  @end
-  {@BREAK}
```

Comment:
I haven't touched the smoke test and the test still passes so I think it's okay.